docs/release-notes/0.9/release-0.9.md

```diff
@@ -0,0 +1,48 @@
+# ML.NET 0.9 Release Notes
+
+Welcome to 2019! For the past 9 months we have been working on adding features and improving [ML.NET](https://aka.ms/mlnet). In the remaining releases before we reach 1.0, we are focusing more on the overall stability of the package. To acheive that we will continue to work on refining the API, increasing test coverage and improving documentation. 0.9 release packs multiple fixes as well as significant clean up to the internal code of the package.
```

Comment:
just some reduction in wording:
For the past 9 months **we have added features and improved upon ml.net**(...). In the remaining release..., **we will focus on** the overall stability of the package. **We will continue to refine the API, increase test coverage and improve documentation.**

If you keep the original wording, acheive needs to be achieve.